### PR TITLE
[What's New] SQLite Plugin

### PR DESCRIPTION
**Pull request by seancdavis**
There are a handful of changes in here:

- New _What's New_ post: _SQLite Plugin_
- What's New posts now uses new SEO component
- Cleanup of some typing conventions
- Quick fix on SendGrid image

I intentionally left this post short, since there are related updates that will drip over the coming weeks, which include:

- app-example-config using SQLite plugin
- Plugin to support app-example-config

### CODE REVIEW

File: pages/whats-new/2021-04-01-sqlite-plugin.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+image: releases/2021-04-01-sqlite-plugin.png
+github: https://github.com/grouparoo/grouparoo/pull/1503
+twitterCard: summary_large_image
+---
+
+Grouparoo [has a new plugin](/docs/installation/plugins): [SQLite](https://www.npmjs.com/package/@grouparoo/sqlite)!
+
+[SQLite](https://www.sqlite.org/index.html) is a lightweight database that holds its data in a single `.sqlite `file on disk. Up until now, Grouparoo has supported SQLite as its application database, but not as a Source or Destination for your customer data.
+
+With this addition, you can now use SQLite to as a [Source](/docs/config/sources/community) — to import your customer data — or a [Destination](/docs/config/destinations/community) – to export your Grouparoo Profiles and Groups.
```
Comment (andyjih) [inline pages/whats-new/2021-04-01-sqlite-plugin.mdx]:
```suggestion
With this addition, you can now use SQLite as a [Source](/docs/config/sources/community) — to import your customer data — or a [Destination](/docs/config/destinations/community) – to export your Grouparoo Profiles and Groups.
```

**Review by andyjih — APPROVED** [review]
Looks good!

File: pages/whats-new/2021-04-01-sqlite-plugin.mdx (hunk 1 above)
Comment (seancdavis) [reply pages/whats-new/2021-04-01-sqlite-plugin.mdx]: Thank you! Nice catch!

**Review by evantahler — APPROVED** [review]
Looks great, and the SEO component is going to pay off for sure!

File: pages/whats-new/[note].tsx [hunk 2]
```diff
@@ -9,38 +9,23 @@ import {
...
-        />
-        <meta property="og:title" content={feature.title} />
-        {feature.description ? (
-          <meta property="og:description" content={feature.description} />
-        ) : null}
-        {feature.image ? (
-          <meta property="og:image" content={feature.image} />
-        ) : null}
-      </Head>
+      <SEO
```
Comment (bleonard) [inline pages/whats-new/[note].tsx:21]: neat. should we use this in other spots?

**Pull request merged**